libs/accounts/recovery-phone/src/lib/recovery-phone.service.ts

```diff
@@ -83,21 +88,26 @@ export class RecoveryPhoneService {
     }
 
     const code = await this.otpCode.generateCode();
-    const msg = await this.smsManager.sendSMS({
-      to: phoneNumber,
-      body: code,
-    });
 
-    if (!this.isSuccessfulSmsSend(msg)) {
-      return false;
-    }
     await this.recoveryPhoneManager.storeUnconfirmed(
       uid,
       code,
       phoneNumber,
       true
     );
-    return true;
+
+    const smsBody = localizedMessageBody
+      ? `${localizedMessageBody.part1} ${code} ${localizedMessageBody.part2}`
+      : `${code}`;
+
+    let msg = {} as MessageInstance;
+
+    msg = await this.smsManager.sendSMS({
+      to: phoneNumber,
+      body: smsBody,
```

Comment:
See comment above about template function, if that was approach was used this would just become

```
body: localizedMessageBody(code)
```